### ISSUE TITLE
resto druid 'upgradedLegendaries' of undefined

### ISSUE THREAD

**Issue by Medivha** [WoWAnalyzer/WoWAnalyzer#2135]
```
TypeError: Cannot read property 'upgradedLegendaries' of undefined
    at t.check (https://wowanalyzer.com/static/js/Druid.abb5413b.chunk.js:1:914894)
    at https://wowanalyzer.com/static/js/Druid.abb5413b.chunk.js:1:901020
    at Array.map (<anonymous>)
    at t.value (https://wowanalyzer.com/static/js/Druid.abb5413b.chunk.js:1:900990)
    at Array.map (<anonymous>)
    at t.value (https://wowanalyzer.com/static/js/Druid.abb5413b.chunk.js:1:904216)
    at t.value (https://wowanalyzer.com/static/js/main.663375a1.js:1:1178450)
    at t.value (https://wowanalyzer.com/static/js/main.663375a1.js:1:1181225)
    at t.value (https://wowanalyzer.com/static/js/main.663375a1.js:1:1182613)
    at Rn (https://wowanalyzer.com/static/js/main.663375a1.js:1:422552)
    at Cn (https://wowanalyzer.com/static/js/main.663375a1.js:1:428160)
    at qn (https://wowanalyzer.com/static/js/main.663375a1.js:1:436909)
    at Zn (https://wowanalyzer.com/static/js/main.663375a1.js:1:437217)
    at da (https://wowanalyzer.com/static/js/main.663375a1.js:1:440601)
    at ca (https://wowanalyzer.com/static/js/main.663375a1.js:1:440110)
    at sa (https://wowanalyzer.com/static/js/main.663375a1.js:1:439936)
    at ra (https://wowanalyzer.com/static/js/main.663375a1.js:1:439308)
    at $n (https://wowanalyzer.com/static/js/main.663375a1.js:1:438667)
    at Object.enqueueSetState (https://wowanalyzer.com/static/js/main.663375a1.js:1:470821)
    at t.r.setState (https://wowanalyzer.com/static/js/main.663375a1.js:1:380212)
    at t.<anonymous> (https://wowanalyzer.com/static/js/main.663375a1.js:1:1159515)
    at a (https://wowanalyzer.com/static/js/main.663375a1.js:1:554303)
    at Generator._invoke (https://wowanalyzer.com/static/js/main.663375a1.js:1:555349)
    at Generator.e.(anonymous function) [as next] (https://wowanalyzer.com/static/js/main.663375a1.js:1:554482)
    at a (https://wowanalyzer.com/static/js/main.663375a1.js:1:1155105)
    at https://wowanalyzer.com/static/js/main.663375a1.js:1:1155199
```
on g'huun norml kill

**Comment by Pewtro** [comment]
Can you give me a link to the log, and I'll fix it right after my raid in a couple of minutes

**Comment by Pewtro** [comment]
Fixed in #2136
